.github/workflows/ci.yml

```diff
@@ -16,7 +16,7 @@ jobs:
           node-version: ${{ matrix.node-version }}
           cache: 'yarn'
       - name: Install modules
-        run: yarn install --immutable
+        run: yarn install --immutable --ignore-scripts
```

Comment:
Hmm, seems like `--immutable` is supported in v2 only and `--ignore-scripts` only in v1 🤔

Reply:
Ohh! We're using version 1 here, seeing that this change had the desired impact. This was renamed from `--frozen-lockfile`, changing it to that!

Reply:
Fixed!